release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -92,6 +92,13 @@ For more information about enabling a feature set during installation, see xref:
 ==== {product-title} on ARM
 {product-title} {product-version} is now supported on ARM architecture-based Azure installer-provisioned infrastructure. AWS Graviton 3 processors are now available for cluster deployments and are also supported on {product-title} 4.11. For more information about instance availability and installation documentation, see xref:../installing/installing-preparing.html#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms]
 
+[id="ocp-4-12-oc-mirror-oci"]
+==== Mirroring Operator images in OCI format with the oc-mirror CLI plug-in (Technology Preview)
```

Comment:
```suggestion
==== Mirroring File Based Catalog (FBC) Operator images in OCI format with the oc-mirror CLI plug-in (Technology Preview)
```

Reason is we need to make sure our users understand we do not support oci format for remote catalogs, or images other than catalogs, just the file based ones.

Reply:
Thanks @sherine-k I can do that. However, we don't mention "file-based catalog" in the title of the main content we link to for this. Should I update that title there as well?

https://53594--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-oci-format_installing-mirroring-disconnected

Reply:
@bergerhoffer  -  Yes the more we mention fbc the better, in reality its only the operator catalog that's in oci format, we don't want to cause any confusion that its used for release, additionalImages etc

Reply:
Thanks! Updating